engine/client/cl_view.c

```diff
@@ -417,6 +421,12 @@ void V_PostRender( void )
 		SCR_RSpeeds();
 		SCR_NetSpeeds();
 		SCR_DrawFPS();
+
+#ifdef STROBE_ENABLED
+		if (STROBE_CORE)
+			STROBE_CORE->STROBE_CORE_FUNC_debughandler(&STROBE_CORE);
```

Comment:
Well... now it looks like something from Windows API...

Reply:
Macros can be extended to make this more pretty but there are already a lot of them.